rviz_rendering/CMakeLists.txt

```diff
@@ -107,41 +110,57 @@ target_link_libraries(rendering_example rviz_rendering Qt5::Widgets)
 #                see:
 # https://stackoverflow.com/questions/41193584/deploy-all-qt-dependencies-when-building#41199492
 if(TARGET Qt5::windeployqt)
-    # execute windeployqt in a tmp directory after build
-    add_custom_command(TARGET rendering_example
-        POST_BUILD
-        COMMAND ${CMAKE_COMMAND} -E remove_directory "${CMAKE_CURRENT_BINARY_DIR}/windeployqt"
-        COMMAND set PATH=%PATH%$<SEMICOLON>${qt5_install_prefix}/bin
-        COMMAND
-          Qt5::windeployqt
-          --dir "${CMAKE_CURRENT_BINARY_DIR}/windeployqt"
-          "$<TARGET_FILE_DIR:rendering_example>/$<TARGET_FILE_NAME:rendering_example>"
```

Comment:
Re-indenting to two space is correct, but the extra indentation for the command's arguments should have stayed. You don't have to fix this, but for the future reference.